test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -64,11 +64,16 @@ Describe "Test-Connection" -tags "CI" {
             { $result = Test-Connection "fakeHost" -Count 1 -Quiet -ErrorAction Stop } |
```

Comment:
@JamesWTruher, your last commit had 2 failures in `PowerShell-CI-linux`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 175
175:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a configuration with multiple resources on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource.

At line:10 char:17
+                 Script s1
+                 ~~~~~~
Undefined DSC resource 'Script'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 195
195:         [Scriptblock]::Create(@"

```